src/coalescenceml/orchestrator/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+## Orchestrator
+
+An orchestrator is a special kind of backend that manages the running of each
+step of the pipeline. Orchestrators administer the actual pipeline runs.
+"""
```

Comment:
I think it would be useful if every major component has something similar to this so that those who don't understand the overall structure of COML (including me :')) gets a better understanding of how it works